index.d.ts

```diff
@@ -0,0 +1,16 @@
+import { Context } from 'aws-lambda';
+
+interface ContextOpts {
+	region: string;
+	account: string;
+	functionName: string;
+	functionVersion: string;
+	memoryLimitInMB: string;
```

Comment:
Add `alias` as option.

Reply:
Ah, I missed alias from the body of the fn - good catch 👍